src/main/java/duke/TaskList.java

```diff
@@ -0,0 +1,104 @@
+package duke;
+
+import java.util.ArrayList;
+
+public class TaskList {
+
+    protected ArrayList<Task> tasks;
+
+    /**
+     * TaskList constructor when there are tasks in file
+     *
+     * @param tasks
+     */
+    public TaskList(ArrayList<Task> tasks) {
+        this.tasks = tasks;
+    }
+
+    /**
+     * TaskList constructor when there are no tasks in file
+     */
+    public TaskList() {
+        this.tasks = new ArrayList<Task>(100);
+    }
+
+    /**
+     * Gives the number of tasks in the list
+     *
+     * @return the number of tasks in the list
+     */
+    public int size() {
```

Comment:
Maybe we can use getSize() instead of size(), it will be easier to read this way.